lib/lbt/bundle/BundleWriter.js

```diff
@@ -25,6 +29,8 @@ class BundleWriter {
 	write(...str) {
 		for ( let i = 0; i < str.length; i++ ) {
 			this.buf += str[i];
+			this.lineOffset += str[i].split(NL).length - 1;
+			this.columnOffset += str[i].length;
```

Comment:
shouldn't this just be the length of the last line if there's any NL in the str and be incremented, when there's no NL?

Reply:
Yes, you're correct, it works today because I just deal with minified, one line files ;)